rust-toolchain.toml

```diff
@@ -2,10 +2,10 @@
 channel = "nightly-2023-02-01"
 components = [
     # https://github.com/rust-lang/rust/issues/72594#issuecomment-633779564
-    "llvm-tools-preview",
+    "llvm-tools",
     # For components/script_plugins, https://github.com/rust-lang/rust/pull/67469
     "rustc-dev",
     "rust-docs",
-    "rustfmt-preview",
+    "rustfmt",
```

Comment:
This wasn't from formatted but resolves warnings from extension.

Reply:
Hrm. I'm a bit nervous about changing these. Why would formatting TOML files require us to use different rustup components?

Reply:
When components where first introduced the got -preview prefix, but now they aren't using prefixes anymore as they are stable. Aliases are still in cargo to not make be breaking change.

Reply:
Thanks for the background. I'd be happy to review that as a separate PR.